lib/tmux-gitbar.sh

```diff
@@ -44,9 +44,13 @@ RESET_FMT="#[fg=default]"
 TMGB_OUTREPO_STATUS=""
 TMGB_OUTREPO_STYLE=""
 
+if [ -z ${TMUX_GITBAR_CONF} ]; then
+  TMUX_GITBAR_CONF=${HOME}/.tmux-gitbar.conf
+fi
+
 # Load the config file
 load_config() {
-  source "${SCRIPT_DIR}/../tmux-gitbar.conf"
+  source "${TMUX_GITBAR_CONF}"
```

Comment:
it seems the `no_extra_output.tcl` integration test fails because the config file is not found, and the string `/home/travis/.tmux-gitbar.conf: No such file or directory\r\`,amongst others, are produced when $PROMPT_COMMAND is invoked.